server/build-plugins.js

```diff
@@ -33,6 +35,47 @@ const getPackageName = name => {
   }
 };
 
+async function installRemotePackages(installPackages) {
```

Comment:
This was in the main function before, but moved here to separate out the code a bit more.